internal/dataplane/kong_client.go

```diff
@@ -439,6 +452,32 @@ func (c *KongClient) Update(ctx context.Context) error {
 	return nil
 }
 
+func (c *KongClient) FetchLastGoodConfiguration(ctx context.Context) error {
+	gatewayClients := c.clientsProvider.GatewayClients()
+	c.logger.Debugf("sending configuration to %d gateway clients", len(gatewayClients))
+
+	var goodKongState *kongstate.KongState
+	for _, client := range gatewayClients {
+		rs, err := dump.Get(ctx, client.AdminAPIClient(), dump.Config{})
+		if err != nil {
+			return err
+		}
+		ks := kongstate.KongRawStateToKongState(rs)
+		status, err := client.AdminAPIClient().Status(ctx)
+		if err != nil {
+			return err
+		}
+		if status.ConfigurationHash != sendconfig.WellKnownInitialHash {
+			// Get the first good one as the one to be used.
+			goodKongState = ks
+		}
+	}
+	if goodKongState != nil {
+		c.lastValidKongState = goodKongState
+	}
```

Comment:
```suggestion
	if goodKongState != nil {
		c.lastValidKongState = goodKongState
		break
	}
```

Reply:
We want to be sure there are no errors with any client here.

Reply:
> EDIT: or maybe not? 🤔 what's your thought on this one? We should simply pick the first good one and not checking the others?

Yeah, I think so 👍

Reply:
Yeah, it makes sense. I'll change it.